### DISCUSSION TITLE
Can't get the Weather feature to work

### DISCUSSION THREAD

**Discussion by zKashim** [valinet/ExplorerPatcher#3755] [Q&A]
I recently upgraded from Windows 10 to Windows 11 and hated the new taskbar. I was using 7+ Taskbar Tweaker for years and since it doesn't work on Windows 11 I had to switch to Windhawk, which I also hated.

Anyway, there was a post on the 7+ Taskbar site suggesting to use ExplorerPatcher to get back the old Windows 10 taskbar functionality. I installed it and instantly fell in love. Everything works great except the Weather widget only shows the People app instead of the actual weather information.

In the EP settings I have Taskbar style set to Windows 10 and Start button style set to Windows 10. In Windows 11 Settings -> Personalization -> Taskbar page I have Widgets enabled. On the Installed apps list I have Microsoft Edge WebView2 Runtime 129.0.279.65 installed. I even tried to manually install the Evergreen WebView2 standalone component, but it says it's already installed. 

I tried restarting File Explorer multiple times, but I never actually saw the weather info. It only ever shows the People icon.

Every other EP feature works perfectly except the Weather widget. What am I doing wrong? Am I missing something here?

Any help would be greatly appreciated.

**Comment by pyrates999** [comment]
Try going through this guide: https://github.com/valinet/ExplorerPatcher/wiki/Weather

Already discussed here: https://github.com/valinet/ExplorerPatcher/issues/3624

**Reply by zKashim** [reply]
Thanks for the reply. That is the "guide" I used, since it's linked right inside the Weather tab of EP. There's nothing on there about troubleshooting or getting the Weather feature to work. It basically says if you have WebView2 installed then it should work, but it doesn't work for me. It's also the page that suggested installing the standalone version of WebView2, which I also tried.

**Comment by pyrates999** [comment]
I think this is the same as this: https://github.com/valinet/ExplorerPatcher/issues/2818

**Reply by zKashim** [reply]
It is not the same. That poster says they're running Open Shell, which I'm not, and they had Weather working, but it stopped. Mine never worked at all. And they said they can't even enable/disable it from EP app window. I can enable it, but it only shows the People window instead of Weather.

**Reply by pyrates999** [reply]
The weather guide specifies what you are to do if you get the people icon: https://github.com/valinet/ExplorerPatcher/wiki/Weather

**Reply by zKashim** [reply]
Again, it just says to manually install WebView2, which I already tried. If you're just going to repeat the same nonsense then please stop replying. Thank you.

**Reply by pyrates999** [reply]
please close this as it is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2818

**Reply by zKashim** [reply]
Dude, what is your problem? You keep replying with useless information. If you have nothing relevant to say just keep your mouth shut.

**Comment by zKashim** [comment]
I found the problem and posting the solution here in case anyone else is having the same issue. I installed EP on two different machines, one running Windows 10 and the other Windows 11. The one running Windows 10 had no problems, but on the other one Windows Security quarantined the file ep_weather_host_stub.dll and that's the reason it wasn't working. As soon as I moved the file out of quarantine and reloaded EP everything was fine.

**Reply by pyrates999** [reply]
It is happening on windows 10 as well.  For me, I've had to exclude the file directly.  Run powershell with elevated privileges to run this:

`Add-MpPreference -ExclusionPath "C:\Program Files\ExplorerPatcher\ep_weather_host_stub.dll"`
